### PR TITLE
Enhancement: Introduce DOCtor-RST config + GithubAction

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: doc/tags/for.rst [hunk 1]
```diff
@@ -86,7 +86,7 @@ The `else` Clause
 If no iteration took place because the sequence was empty, you can render a
 replacement block by using ``else``:
 
-.. code-block:: twig
+.. code-block:: html+wig
```
Comment (derrabus) [inline doc/tags/for.rst]: This does not look right.

File: doc/tags/macro.rst [hunk 2]
```diff
@@ -57,15 +57,15 @@ the ``forms`` local variable.
 
 The macros can then be called at will in the *current* template:
 
-.. code-block:: twig
+.. code-block:: html+wig
```
Comment (derrabus) [inline doc/tags/macro.rst]: Same.

**Review by derrabus — CHANGES_REQUESTED** [review]

File: doc/tags/macro.rst (hunk 2 above)
Comment (OskarStark) [reply doc/tags/macro.rst:60]:
```suggestion
.. code-block:: html+twig
```

File: doc/tags/for.rst (hunk 1 above)
Comment (OskarStark) [reply doc/tags/for.rst]: Thanks, fixed!

File: doc/tags/macro.rst (hunk 2 above)
Comment (OskarStark) [reply doc/tags/macro.rst]: Thanks fixed

File: doc/.doctor-rst.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        major_version: 3
+
+    versionadded_directive_min_version:
+        min_version: '3.0'
+
+    deprecated_directive_major_version:
+        major_version: 3
+
+    deprecated_directive_min_version:
+        min_version: '3.0'
```
Comment (OskarStark) [inline doc/.doctor-rst.yaml]:
```suggestion
        min_version: '3.0'

```

**Comment by fabpot** [comment]
Thank you @OskarStark.

**Pull request merged**